tests/test_smoke.py

```diff
@@ -1144,7 +1144,7 @@ def test_gcp_stale_job_manual_restart():
             # Ensure the skylet updated the stale job status.
             _get_cmd_wait_until_job_status_contains_without_matching_job(
                 cluster_name=name,
-                job_status=[JobStatus.FAILED.value],
+                job_status=[JobStatus.FAILED],
```

Comment:
For this kind of hot fixes, we may want to include it in master and cherry pick it?

Reply:
It's due to a merge conflict. The master branch's value is `FAILED_DRIVER`, which does not exist in version 0.7.1 but is correct in the master branch.